<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswdWidget</class>
 <widget class="QWidget" name="ChangePasswdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SubmitButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="OldPasswdLabel">
        <property name="text">
         <string>  Old Passwd   ：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="OldPasswdLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="NEWPasswdLabel">
        <property name="text">
         <string>   New Passwd：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NewPasswdLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="CheckPasswdLabel">
        <property name="text">
         <string>Confirm Passwd:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="CheckPasswdLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
